shared/src/commonMain/kotlin/io/customer/shared/common/TypeAliases.kt

```diff
@@ -1,6 +1,7 @@
 package io.customer.shared.common
 
 typealias CustomAttributesCompat = Map<String, Any>
+typealias QueueTaskResult = Boolean
```

Comment:
Since we don't need to pass a lot of data in task results yet, I used a typealias here to visually give the idea of what `Boolean` value indicates, and created extension methods to avoid ambiguities. e.g. `QueueTaskResult.Success()`, `TaskResultListener.success()`. Does this look a bit over-engineered? If yes, would you suggest to use `Boolean` directly or wrap `Boolean` inside a `QueueTaskResult` class to keep more flexibility?

Reply:
Yeah, agreed. Refactoring anything in internal code is comperatively much easier. I'll replace all usages of `QueueTaskResult` with `Boolean` directly.